app/models/manageiq/providers/ibm_cloud/inventory/collector/object_storage.rb

```diff
@@ -0,0 +1,7 @@
+class ManageIQ::Providers::IbmCloud::Inventory::Collector::ObjectStorage < ManageIQ::Providers::IbmCloud::Inventory::Collector
+  require_nested :StorageManager
+
+  def connection
+    @connection ||= manager.connect
+  end
+end
```

Comment:
Seems the bot isn't running on this repo otherwise it would have picked up on this?

Reply:
We do it in PowerVS too. Do you mean it should be done in a different way?